mappings/net/minecraft/client/InteractionManager.mapping

```diff
@@ -1,17 +1,60 @@
 CLASS net/minecraft/class_504 net/minecraft/client/InteractionManager
 	FIELD field_2104 minecraft Lnet/minecraft/client/Minecraft;
+	FIELD field_2105 noTick Z
 	METHOD <init> (Lnet/minecraft/client/Minecraft;)V
 		ARG 1 minecraft
+	METHOD method_1705 resetBlockBreaking ()V
+	METHOD method_1706 update (F)V
+	METHOD method_1707 interactBlock (IIII)V
+		ARG 1 x
+		ARG 2 y
+		ARG 3 z
+		ARG 4 direction
 	METHOD method_1708 clickSlot (IIIZLnet/minecraft/class_54;)Lnet/minecraft/class_31;
 		ARG 1 windowId
```

Comment:
While this isnt part of the PR, it feels fitting to correct this to syncId while we are here